src/config/app.spec.ts

```diff
@@ -47,7 +56,41 @@ describe('config/app', () => {
                 phpIni                        : [ 'error_reporting=E_ALL' ],
                 ignorePhpPlatformRequirements : {},
                 additionalComposerArguments   : [],
+                backwardCompatibilityCheck    : false,
+                targetReference               : null,
+            });
+        });
+
+        it('should detect GITHUB_BASE_REF', () => {
+            const environment = process.env;
+
+            configDotenv({path: `${roaveBackwardCompatibilityPath}/test.env`});
```

Comment:
Fancy solution :O

Reply:
seems to be kinda standard for JEST, had only a quick search on google and it was the first suggestion once it comes to testing stuff including environment 😅